### PR TITLE
Update download location of verrazzano release bundles for private registry setup

### PR DESCRIPTION
**Pull request by bhatpmk**
This change updates the location from where the Verrazzano release bundles can be downloaded, in the documentation for setting up Private Registry.

### CODE REVIEW

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md [hunk 1]
```diff
@@ -75,26 +75,44 @@ Set up a private registry using the following instructions, depending on your di
...
+        * To download the ZIP file directly, select the file link in the list.
+        * To download the ZIP file using `Oracle Download Manager`, click **Download** and run the `Oracle Download Manager` executable.
 
-     * To download the ZIP file directly, select the file link in the list.
-     * To download the ZIP file using `Oracle Download Manager`, click **Download** and run the `Oracle Download Manager` executable.    
+    * Download the Verrazzano ZIP file from My Oracle Support for cumulative patches.
+
+        a. In your browser, go to [My Oracle Support](https://support.oracle.com/) and log in with your credentials.
+
+        b. Select `Patches & Updates` tab at the top of the `MY ORACLE SUPPORT` page.
```
Comment (lake-of-dreams) [inline content/en/docs/setup/private-registry/private-registry-full-distribution.md:103]: I find it confusing that this page is called `MY SUPPORT PAGE` , can't this be just `tab at the top of the page`?

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md [hunk 2]
```diff
@@ -75,26 +75,44 @@ Set up a private registry using the following instructions, depending on your di
...
+
+        c. In the `Patch Search` panel, click the link `Product or Family (Advanced)`.
+
+        d. In the search bar for `Product is`, enter `Oracle Verrazzano Enterprise Container Platform`.
+
+        e. The above step populates the available releases for Verrazzano in the dropdown box `Release is`. Select the desired release(s) and click **Search**.
+
+        f. Click the link for the desired patch, under the `Patch Name` and click **Download**.
```
Comment (lake-of-dreams) [inline content/en/docs/setup/private-registry/private-registry-full-distribution.md:111]: I guess may be good to specify that a `New panel with Patch Advanced Search Results will open listing all the patches for the release`

Comment (lake-of-dreams) [inline content/en/docs/setup/private-registry/private-registry-full-distribution.md:111]: If I don't click `Download` , it still takes me to Patch detail page, maybe worth mentioning that

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md (hunk 1 above)
Comment (MelwynPaul) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md:103]: Or just - 
b. Select the `Patches & Updates` tab.

Comment (bhatpmk) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md]: Done

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md (hunk 2 above)
Comment (bhatpmk) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md]: Done

Comment (bhatpmk) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md]: I have just removed that particular download instruction.

**Review by david-cabelus-oracle — APPROVED** [review]

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md [hunk 3]
```diff
@@ -75,26 +75,44 @@ Set up a private registry using the following instructions, depending on your di
...
+        a. In your browser, go to the [Oracle Software Delivery Cloud](https://edelivery.oracle.com) and log in with your credentials.
 
-    b. In the drop-down menu preceding the search bar, select **All Categories**.
+        b. In the drop-down menu preceding the search bar, select **All Categories**.
 
-    c. In the search bar, enter `Verrazzano Enterprise Container Platform` and click **Search**.
+        c. In the search bar, enter `Verrazzano Enterprise Container Platform` and click **Search**.
 
-    d. Select the `DLP: Oracle Verrazzano Enterprise Edition {{<download_package_version>}}` link.  This will add it to your download queue.
+        d. Select the `REL: Oracle Verrazzano Enterprise Edition {{<download_package_version>}}` link.  This will add it to your download queue.
```
Comment (MelwynPaul) [inline content/en/docs/setup/private-registry/private-registry-full-distribution.md:87]: Verified the procedure (till before installing VZ) - Looks good.

Very minor comment.
Please check - 
For 1.3.0.0.0, I see the result as "REL: Verrazzano Enterprise Container Platform 1.3.0.0.0" 
No "Oracle" tag.

Although, for other products 'Oracle' is present. 
On the Oracle Software Delivery Cloud page, should VZ also have 'Oracle'?

**Review by MelwynPaul — COMMENTED** [review]
Content looks good. 
Just one UI-related comment.

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md (hunk 3 above)
Comment (bhatpmk) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md]: @MelwynPaul Thanks for finding this, corrected it now.

Comment (MelwynPaul) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md]: :+1:

**Review by MelwynPaul — APPROVED** [review]
Thank you.
Looks good to me.

**Review by rosemarymarano — COMMENTED** [review]
Sorry I missed these UI navigation corrections needed previously. Please fix them.

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md (hunk 3 above)
Comment (bhatpmk) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md]: Done

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md [hunk 4]
```diff
@@ -75,26 +75,44 @@ Set up a private registry using the following instructions, depending on your di
...
+
+        a. In your browser, go to [My Oracle Support](https://support.oracle.com/) and log in with your credentials.
+
+        b. Select the `Patches & Updates` tab.
+
+        c. In the `Patch Search` panel, click the link `Product or Family (Advanced)`.
+
+        d. In the search bar for `Product is`, enter `Oracle Verrazzano Enterprise Container Platform`.
+
+        e. The above step populates the available releases for Verrazzano in the dropdown box `Release is`. Select the desired release(s) and click **Search**.
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md]: Updated as per the suggestion

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md [hunk 5]
```diff
@@ -75,26 +75,44 @@ Set up a private registry using the following instructions, depending on your di
...
 
-     * To download the ZIP file directly, select the file link in the list.
-     * To download the ZIP file using `Oracle Download Manager`, click **Download** and run the `Oracle Download Manager` executable.    
+    * Download the Verrazzano ZIP file from My Oracle Support for cumulative patches.
+
+        a. In your browser, go to [My Oracle Support](https://support.oracle.com/) and log in with your credentials.
+
+        b. Select the `Patches & Updates` tab.
+
+        c. In the `Patch Search` panel, click the link `Product or Family (Advanced)`.
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md]: Done

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md [hunk 6]
```diff
@@ -75,26 +75,44 @@ Set up a private registry using the following instructions, depending on your di
...
+
+        d. In the search bar for `Product is`, enter `Oracle Verrazzano Enterprise Container Platform`.
+
+        e. The above step populates the available releases for Verrazzano in the dropdown box `Release is`. Select the desired release(s) and click **Search**.
+
+        f. A new panel with `Patch Advanced Search Results` will open listing all the patches for the release. Click the link for the desired patch, under the `Patch Name`.
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md]: Done

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md [hunk 7]
```diff
@@ -75,26 +75,44 @@ Set up a private registry using the following instructions, depending on your di
...
+
+        a. In your browser, go to [My Oracle Support](https://support.oracle.com/) and log in with your credentials.
+
+        b. Select the `Patches & Updates` tab.
+
+        c. In the `Patch Search` panel, select the link `Product or Family (Advanced)`.
+
+        d. In the search bar for `Product is`, enter `Oracle Verrazzano Enterprise Container Platform`.
+
+        e. The previous step populates the available releases for Verrazzano in the drop-down box `Release is`. Select the desired release(s) and click **Search**.
```
Comment (rosemarymarano) [inline content/en/docs/setup/private-registry/private-registry-full-distribution.md:109]: drop-down box -> drop-down menu

**Review by rosemarymarano — APPROVED** [review]
One minor one left, otherwise ok.

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (bhatpmk) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md]: Done

**Pull request merged**